include/session.php

```diff
@@ -30,7 +30,7 @@ function ref_session_read($id) {
 
 	$r = q("SELECT `data` FROM `session` WHERE `sid`= '%s'", dbesc($id));
 
-	if (dbm::is_result($r)) {
+	if(dbm::is_result($r)) {
```

Comment:
@Quix0r Yeah, there is much work to do. We agreed some time before that we convert all lines that we are changing according to PSR-2. We are still a little bit unsure about some details but agree in most things.